### PR TITLE
Differences in vote column in mem_accounts while syncing from scratch - Closes #2420

### LINKED ISSUE #2420: Differences in vote column in mem_accounts while syncing from scratch
### Expected behavior
Use the logic which was part of the protocol and persisted in the blockchain.

### Actual behavior
When a `1.1.0` node sync from scratch with test network, the `1.1.0` node failed to sync due to differences in `mem_accounts` table (`vote` column) leading to change in delegate list order, the change in order of delegate resulting in unrecoverable fork3, here is the logs for more information.
```
[inf] 2018-09-24 15:58:54 | Verify->verifyBlock succeeded for block 14304594437763848243 at height 809518.
[ERR] 2018-09-24 15:58:54 | Expected generator: e156529dae0f0f586539ff2f8eebc1b735e67687f6d2fff45ea01d7dbd71533e Received generator: f4a319a653a3d5f3bbe54fabd563d9444e05a776eec005bcf961c31ec6f89c15
[inf] 2018-09-24 15:58:54 | Fork - {"delegate":"f4a319a653a3d5f3bbe54fabd563d9444e05a776eec005bcf961c31ec6f89c15","block":{"id":"14304594437763848243","timestamp":14261020,"height":809518,"previousBlock":"781869470176582773"},"cause":3}
[inf] 2018-09-24 15:58:54 | Banned peer - 46.101.22.213:7001
[ERR] 2018-09-24 15:58:54 | Error loading blocks: Failed to verify slot: 1426102
[ERR] 2018-09-24 15:58:54 | Failed to load blocks from: 46.101.22.213:7001
[inf] 2018-09-24 15:58:54 | Looking for common block with: 178.62.106.202:7001
[inf] 2018-09-24 15:58:54 | Releasing enqueued broadcasts
[inf] 2018-09-24 15:58:54 | Queue empty
[inf] 2018-09-24 15:58:54 | Found common block: 781869470176582773 with: 178.62.106.202:7001
[inf] 2018-09-24 15:58:54 | Loading blocks from: 178.62.106.202:7001
[inf] 2018-09-24 15:58:54 | Verify->verifyBlock succeeded for block 14304594437763848243 at height 809518.
[ERR] 2018-09-24 15:58:54 | Expected generator: e156529dae0f0f586539ff2f8eebc1b735e67687f6d2fff45ea01d7dbd71533e Received generator: f4a319a653a3d5f3bbe54fabd563d9444e05a776eec005bcf961c31ec6f89c15
[inf] 2018-09-24 15:58:54 | Fork - {"delegate":"f4a319a653a3d5f3bbe54fabd563d9444e05a776eec005bcf961c31ec6f89c15","block":{"id":"14304594437763848243","timestamp":14261020,"height":809518,"previousBlock":"781869470176582773"},"cause":3}
[inf] 2018-09-24 15:58:54 | Banned peer - 178.62.106.202:7001
[ERR] 2018-09-24 15:58:54 | Error loading blocks: Failed to verify slot: 1426102
[ERR] 2018-09-24 15:58:54 | Failed to load blocks from: 178.62.106.202:7001
```

The `vote` column in `mem_accounts` is impacted due to the changes https://github.com/LiskHQ/lisk/issues/716 introduced in `1.1.0`, before this change there was a difference of 1 beddow due to the way we handled javascript number, after the fixes in this PR https://github.com/LiskHQ/lisk/pull/2187 the issue was fixed, however this changes will invalidate the blockchain.
### Steps to reproduce
Run `1.1.0` node and sync from scratch with the test network
### Which version(s) does this affect? (Environment, OS, etc...)
1.1.0

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?
Node failed to sync due to differences in mem_accounts table (vote column) leading to change in delegate list order, the change in order of delegate resulting in unrecoverable fork3

### How did I fix it?
Reverted the round logic, matching to `1.0.0`

### How to test it?
Sync from the network of `1.0.0`

### Review checklist

* The PR resolves #2420
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: config/testnet/exceptions.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ module.exports = {
 		'16272500600161825502', // 336424
 		'17197328760149985951', // 341635
 		'18231026627962552928', // 917323
+		'13238495338830587861', // 1305933
```
Comment (MaciejBaj) [inline config/testnet/exceptions.js]: That's due to #2434

**Comment by MaciejBaj** [comment]
This PR will re-open #216

File: config/testnet/exceptions.js (hunk 1 above)
Comment (4miners) [inline config/testnet/exceptions.js]: According to changes made in https://github.com/LiskHQ/lisk/pull/2442/files this exception is no longer a valid one.

File: logic/round.js [hunk 2]
```diff
@@ -146,16 +146,33 @@ class Round {
 		const self = this;
 
 		return self.getVotes(self.scope.round).then(votes => {
-			const queries = votes.map(vote =>
-				self.t.rounds.updateVotes(
+			const queries = votes.map(vote => {
```
Comment (4miners) [inline logic/round.js]: This change was here only for debug purposes, is not needed.

File: modules/accounts.js [hunk 3]
```diff
@@ -104,12 +105,31 @@ Accounts.prototype.generateAddressByPublicKey = function(publicKey) {
...
  */
 Accounts.prototype.getAccount = function(filter, fields, cb, tx) {
+	const publicKey = filter.publicKey;
 	if (filter.publicKey) {
 		filter.address = self.generateAddressByPublicKey(filter.publicKey);
 		delete filter.publicKey;
 	}
 
-	library.logic.account.get(filter, fields, cb, tx);
+	if (typeof fields === 'function') {
```
Comment (4miners) [inline modules/accounts.js]: Same here, it's not needed to be backported, we collected all the info and this piece of code only adds complexity to the codebase.

File: config/testnet/exceptions.js (hunk 1 above)
Comment (nazarhussain) [reply config/testnet/exceptions.js]: @SargeKhan discussed in the morning, this exception is required. @SargeKhan Can you confirm.

Comment (MaciejBaj) [reply config/testnet/exceptions.js]: Ok, let's open a separate PR to add a valid exception connected with fix #2434

File: modules/blocks/utils.js [hunk 4]
```diff
@@ -409,12 +409,16 @@ Utils.prototype.aggregateBlocksReward = function(filter, cb) {
...
+				(filter.start - constants.epochTime.getTime()) / 1000
+			);
 			params.start = params.start.toFixed();
 		}
 
 		if (filter.end !== undefined) {
-			params.end = Math.floor((filter.end - constants.epochTime.getTime()) / 1000);
+			params.end = Math.floor(
+				(filter.end - constants.epochTime.getTime()) / 1000
+			);
```
Comment (nazarhussain) [inline modules/blocks/utils.js]: @4miners Why you introduced this change? Any idea?

Comment (nazarhussain) [reply modules/blocks/utils.js]: Please ignore, its not a change. just formatting from prettier.

**Pull request merged**